client/wcshipping-migration-admin-notice.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import '../assets/stylesheets/migration_to_wcshipping_admin_notice.scss';
+import FeatureAnnouncement from 'components/migration/feature-announcement';
+import ShippingLabel from 'wcs-client/apps/shipping-label';
+import { setNonce, setBaseURL } from 'wcs-client/api/request';
+
+const container = document.getElementById('wcst_wcshipping_migration_admin_notice_feature_announcement');
+const args = container.dataset.args && JSON.parse( container.dataset.args ) || {};
+setNonce( args.nonce );
+setBaseURL( args.baseURL );
+const ShippingLabelStore = ShippingLabel(args);
+const store = createStore(ShippingLabelStore.getReducer(), ShippingLabelStore.getInitialState());
+
+const wcstWCShippingMigrationNoticeButton = document.getElementById('wcst-wcshipping-migration-notice__click');
+const wcstMigrationNoticeDimissButton = document.querySelector('.wcst-wcshipping-migration-notice button.notice-dismiss');
+
+// Helper function to set simple name=>value cookie in days.
+const setCookie = (name, value, days) => {
```

Comment:
[Optional] You could use `window.wpCookies`  to set cookies.

Reply:
Thanks. I like this suggestion. Replaced it with the wp's one bd141b6c